build-plugin/action.yml

```diff
@@ -81,6 +81,29 @@ runs:
         printf "plugin-id=%s\n" "${{ steps.package-plugin.outputs.plugin-id }}"
         printf "plugin-version=%s\n" "${{ steps.package-plugin.outputs.plugin-version }}"
 
+    - name: Checkout code
+      if: ${{ env.USE_CHANGELOG_GENERATOR }}
```

Comment:
Wouldn't it be better to have an input instead of an environment variable to toggle this behavior? Similar to the other  configurable options in the `inputs` section